mesheryctl/internal/cli/root/filter/list.go

```diff
@@ -54,7 +55,21 @@ mesheryctl exp filter list
 		if err != nil {
 			return err
 		}
-
+		prov := models.RemoteProvider{}
+		tokenPath := utils.AuthConfigFile
+		file, err := os.ReadFile(tokenPath)
+		if err != nil {
+			return err
+		}
+		jwtToken := map[string]string{}
+		errCheck := json.Unmarshal(file, &jwtToken)
+		if errCheck != nil {
+			return ErrUnmarshal(errCheck)
+		}
+		_, invalid := prov.VerifyToken(jwtToken["token"])
+		if invalid != nil {
+			return ErrInvalidAuthToken()
+		}
```

Comment:
@alphaX86 @abdullah1308 @suhail34 where is our standard function for this that is reused across all commands?

Reply:
I see that when the token expires the same errors are with other subcommands too

Reply:
I didn't saw any standard function in mesheryctl folder that does verifytoken if there are please do point me